apps/server/src/modules/tool/common/domain/tool-configuration-status.ts

```diff
@@ -0,0 +1,19 @@
+export class ToolConfigurationStatus {
+	latest: boolean;
+
+	isDisabled: boolean;
+
+	isOutdatedOnScopeSchool: boolean;
+
+	isOutdatedOnScopeContext: boolean;
+
+	isUnkown: boolean;
```

Comment:
Please remove is unknown. Since we dont care about backward incompatible changes